sasl/base/src/main/java/org/wildfly/security/sasl/util/SaslMechanismInformation.java

```diff
@@ -55,6 +55,9 @@
  */
 public final class SaslMechanismInformation {
 
+    /**
+     *  A Collection of Sasl Mechanism names.
```

Comment:
@fazer1929 Thank you for the PR. Just a minor nitpick from me, the word collection in java makes it seem like it is a [Collection](https://docs.oracle.com/javase/8/docs/api/java/util/Collection.html), so I would rephrase it

Reply:
Got it, will change it

Reply:
sorry for that 😅. I have updated the PR